src/pointer-analysis/Makefile

```diff
@@ -5,7 +5,8 @@ SRC = value_set.cpp goto_program_dereference.cpp value_set_analysis.cpp \
       value_set_analysis_fivr.cpp value_set_fivr.cpp value_set_domain_fivr.cpp \
       value_set_analysis_fivrns.cpp value_set_fivrns.cpp \
       value_set_domain_fivrns.cpp value_set_dereference.cpp \
-      dereference_callback.cpp
+      dereference_callback.cpp \
+      dynamic_object_name.cpp
```

Comment:
Put a trailing backslash after this then on the following line add `# No more source files`